### DISCUSSION TITLE
Little Issue

### DISCUSSION THREAD

**Discussion by JCastreau** [chromiecraft/chromiecraft#4065] [General]
Hi there cromie team. I have a problem whit 3 of my characters as soon as i port to Exodar or Silvermoon my client do a fatal/critical error and i can t log on them anymore. I am using the original cromie client US based one. I only play whit the cromie client on cromie server. I disable/deleted all my addons and deleted the WDB WTF folders as well ,it seem like its a map error . CMap::LoadWdt() failed World\Maps\Expansion01\Expansion01.wdt 

Hope you can help me and thanks ! Sorry for my bad spelling and punctuations.

**Comment by 55Honey** [comment]
Hi @JCastreau ,
thanks for reaching out. I'm sorry there's a problem with your client.
My best guess is that your client files got corrupted. I suggest you re-download the client, overwrite your existing one and try again.
